test/integration/steps/utilities/utils.py

```diff
@@ -415,6 +415,48 @@ def check_service_history_change_type(service_id: str, change_type: str):
         return "No changes have been made"
 
 
+def check_service_history_specified(service_id: str):
+    service_history = get_service_history(service_id)
+    specified_open_times = service_history[list(service_history.keys())[0]]["new"]["cmsopentimespecified"]
+    return specified_open_times
+
+
+def convert_specified_opening(specified_date, closed_status=False):
```

Comment:
Please can you add a one line description of what this function converts,  as well as type hinting the function would be appreciated.